core/shared/lib/showdown/extensions/ghostimagepreview.js

```diff
@@ -1,5 +1,4 @@
 /* jshint node:true, browser:true */
```

Comment:
I wouldn't worry about it - as soon as this one is merged I have an incoming PR to fix this properly.